File: vowelCount.js
function getCount(str) {
    var count = 0;
    let vowels = ['a','e','i','o','u']
    var i = 0;
    var j = 0;
    
    for(i = 0; i < str.length; i++){
      for(j = 0; j < vowels.length; j++){
        if(str[i]==vowels[j]){
          count++
        }
        else{
          continue
        }
      }
    }
    return count;
  }

File: vowelCount.py
def get_count(s):
    count = 0
    i = 0
    j = 0
    vowels = ['a','e','i','o','u']
    
    for i in range(len(s)):
        for j in range(len(vowels)):
            if s[i] == vowels[j]:
                count = count + 1
            else:
                continue
    
    return count
pass